src/TNetwork.cpp

```diff
@@ -322,12 +326,30 @@ std::shared_ptr<TClient> TNetwork::Authentication(TConnection&& RawConnection) {
     }
 
     if (mServer.ClientCount() < size_t(Application::Settings.MaxPlayers)) {
+
+        if(!Application::Settings.Password.empty()) { // ask password
+            if(!TCPSend(*Client, StringToVector("S"))) {
```

Comment:
So `S` now is a *request* for password, right? We say `A` for "youre good so far", and then `S` if there's a password request? I'm just making sure this works how I read it

Reply:
yes